app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -101,6 +101,21 @@ def max_memory
       16.gigabytes
     end
 
+    def use_v4=(value)
+      value = case value
```

Comment:
Isn't `value = [true, '1'].include?(value)` equivalent but shorter? Or is that less readable?

Reply:
Honestly, this option haven't occurred to me when writing this. The current variant seems a bit more explicit, but I would prefer this way mostly to not having to wait to tests to finish again, especially as today's the 1.18 branching day and we would like to get this in to the release.